src/components/toggle/Toggle.tsx

```diff
@@ -18,6 +22,8 @@ export interface ToggleProps extends Omit<BoxProps, 'className' | 'size'> {
   label?: ReactNode;
   onChange?: (event: ChangeEvent<HTMLInputElement>) => void;
   size?: Exclude<typeof SIZES[number], 'tiny' | 'fullscreen' | 'smallest' | 'hero'>;
+  tooltip?: ReactNode;
+  tooltipProps?: Record<string, any>;
```

Comment:
Should be the allowed tooltip prop types.

Reply:
I dont understand

Reply:
I think Faraz means that this should basically be what's currently in `TooltippedComponent.propTypes` (and I agree)

I guess that means ideally `Tooltip` is also converted to typescript

Reply:
`TooltipProps` from the tooltip component.